File: LottoPlayer/app/src/main/java/com/im/daeseong/lottoplayer/Util/loadUrlTask.java
package com.im.daeseong.lottoplayer.Util;

import android.os.AsyncTask;
import android.webkit.WebView;

public class loadUrlTask extends AsyncTask<String, Void, String>  {

    private final WebView wvWebView;

    public loadUrlTask(WebView wvWebView) {
        this.wvWebView = wvWebView;
    }

    @Override
    protected void onPreExecute() {
        super.onPreExecute();
    }

    protected String doInBackground(String... params) {
        //SystemClock.sleep(1000);
        return params[0];
    }

    @Override
    protected void onPostExecute(String result) {
        wvWebView.loadUrl(result);
    }

}

File: LottoPlayer(kotlin)/app/src/main/java/com/daeseong/lottoplayer/Util/loadUrlTask.kt
package com.daeseong.lottoplayer.Util

import android.os.AsyncTask
import android.os.SystemClock
import android.webkit.WebView

class LoadUrlTask(private val webView: WebView) : AsyncTask<String?, Void?, String>() {

    override fun onPreExecute() {
        super.onPreExecute()
    }

    override fun doInBackground(vararg params: String?): String? {
        SystemClock.sleep(1000)
        return params.getOrNull(0)
    }

    override fun onPostExecute(result: String?) {
        result?.let {
            webView.loadUrl(it)
        }
    }
}
